tls/s2n_connection.c

```diff
@@ -1124,3 +1114,14 @@ const uint8_t *s2n_connection_get_sct_list(struct s2n_connection *conn, uint32_t
     *length = conn->ct_response.size;
     return conn->ct_response.data;
 }
+
+int s2n_connection_is_client_auth_enabled(const struct s2n_connection *s2n_connection)
+{
+    s2n_cert_auth_type auth_type = s2n_connection->config->client_cert_auth_type;
+
+    if (s2n_connection->client_cert_auth_type_overridden) {
+        auth_type = s2n_connection->client_cert_auth_type;
```

Comment:
Just `s2n_connection_get_client_auth_type` to avoid duplicating logic?

Reply:
Done.